Source/WebCore/css/CSSPropertyInitialValues.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2016-2025 Apple Inc. All rights reserved.
```

Comment:
I think this code is all from 2023 and newer:
```suggestion
 * Copyright (C) 2023-2025 Apple Inc. All rights reserved.
```